packages/x-components/src/x-modules/related-tags/wiring.ts

```diff
@@ -124,5 +134,9 @@ export const relatedTagsWiring = createWiring({
   },
   UserClickedOutOfMainModal: {
     clearRelatedTagsQuery
+  },
+  UserAcceptedAQueryPreview: {
+    setRelatedTagsQueryPreview,
+    clearSelectedRelatedTags
```

Comment:
We should add here a wiring `setRelatedTagsParamsFromQueryPreview` to commit the params from the query preview, now we are sending the global extraParams instead of the selected queryPreview ones in the request